packages/components/src/styles/components/app-header.scss

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APP-HEADER
+//
+
+@use "../mixins/interactive-dark-theme" as *;
+
+// ----------------
+// Q: Create tokens?
+$app-header-home-link-padding: 4px;
+
+// Breakpoints WIP: START ----------------------------
+
+// Break points
+$tablet-min-width: 768px;
+$desktop-min-width: 1024px;
+
+// base styles
+.hds-app-header__menu {
+  display: inherit; // may be over-riddden by queries
+  gap: inherit;
+}
+
+// Media queries
+// -------------
+.hds-app-header {
+  // Mobile/small and tablet/medium compound view: applies at tablet min-width and below
+  @media (max-width: #{$desktop-min-width - 1}) {
+    &:not(.hds-app-header__menu-is-open) {
+      .hds-app-header__menu {
+        // dropdown menu is hidden when isOpen = false
+        display: none;
+      }
+    }
+
+    // Display content as dropdown menu in mobile/tablet view
+    .hds-app-header__menu {
+      position: absolute;
+      top: calc(100% + 1px); // We add 1px to account for the bottom border
+      right: 0;
+      left: 0;
+      z-index: 5; // TODO: determine correct z-index to use
+      display: flex;
+      flex-wrap: wrap;
+      // height: calc(100vh - 52px); // it should take the full height of the viewport minus the header height
+      padding: 16px;
+      background: var(--token-color-palette-neutral-700);
+    }
+
+    // Force nested dropdown content to align left in mobile/tablet view
+    .hds-dropdown__content {
+      left: 0;
+    }
+
+    // Hide :globalAfter content in mobile/tablet view
+    .hds-app-header__global-items-mobile-view {
+      display: none;
+    }
+
+    // Since the same component is used for both mobile/tablet and desktop views, we need to force the width to 100% in mobile view
+    .hds-app-header__global-items-desktop-view {
+      &,
+      .hds-dropdown,
+      .hds-dropdown-toggle-button {
```

Comment:
Should the list of items in the Dropdown also be set to width 100%? I mocked [this up quickly](https://www.figma.com/design/noyY6dUMDYjmySpHcMjhkN/branch/5RLNe1engBID42LhAm75NN/HDS-Components?m=auto&node-id=60962-22032&t=ptWRQcJDDMh0jOFM-1) for reference

Reply:
Also asking because in the `AppFrame` showcase I guess I would expect to see a more realistic example where the mobile menu occupies the entire height.

Reply:
I'm worried about trying to override/hack too many styles but I can see...

(Actually don't know if the way I'm forcing the width now is the best way to handle this...)

Reply:
Jory and I discussed this a bit more and we are leaving the width as is for now (at least for phase 1).